http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/KubernetesAppConfig.scala

```diff
@@ -74,4 +75,19 @@ final case class CoaAppConfig(chartName: ChartName,
 ) extends KubernetesAppConfig {
   override lazy val kubernetesServices: List[KubernetesService] = services.map(s => KubernetesService(ServiceId(-1), s))
   override val serviceAccountName = ServiceAccountName(ksaName.value)
+  def coaServices: Set[CoaService] = services
+    .map(_.name)
+    .collect {
+      case ServiceName("cbas") | ServiceName("cbas-ui") => Cbas
+      case ServiceName("wds")                           => Wds
+      case ServiceName("cromwell")                      => Cromwell
+    }
+    .toSet
+}
+
+sealed trait CoaService
+object CoaService {
+  final case object Cbas extends CoaService
+  final case object Wds extends CoaService
+  final case object Cromwell extends CoaService
```

Comment:
Wondering if we should include TES here as well. I assume there will be a time when we are deploying TES in dev, but not ready to deploy to prod yet. We could also just lump that in with Cromwell.

Reply:
Yeah it might make sense to track TES here, I could imagine wanting to conditionally enable/disable it. I can add an enum value, it won't be used yet though.

Reply:
Just as an FYI, I commented out TES in https://github.com/broadinstitute/cromwhelm/pull/168

Let me know if it would be helpful to uncomment and surround the TES Deployment template with similar `enabled` flags :D

Reply:
Yeah -- I think fine to leave it commented, as it didn't work anyway. We can re-add TES to cromwhelm when we're ready to support it.